crates/common/src/fmt/ui.rs

```diff
@@ -489,6 +489,7 @@ sha3Uncles           {}
 size                 {}
 stateRoot            {}
 timestamp            {}
+time                 {}
```

Comment:
maybe a bit misleading to have its own row, I would put it after timestamp in parentheses

Reply:
I would change it. 
Is it ok to be like
```
timestamp            {}\t{}
```
Or any delimiter recommended?